config/locales/activerecord.yml

```diff
@@ -1,5 +1,10 @@
+dictionary:
+  job_title_blank: &job_title_blank Enter a job title
```

Comment:
Although I believe it's a good practice using aliases I'm wondering if we could rewrite these in a more organised way. something like ```dictionary.errors.messages.job_title.blank: Enter a job title ```